src/test/java/io/airlift/units/TestDataSizeValidator.java

```diff
@@ -34,7 +34,13 @@
 
 public class TestDataSizeValidator
 {
-    private static final Validator VALIDATOR = Validation.byProvider(ApacheValidationProvider.class).configure().buildValidatorFactory().getValidator();
+    private static final Validator VALIDATOR = Validation.byProvider(HibernateValidator.class)
+            .configure()
+            // Disable classpath scanning for configuration for security reasons
+            .ignoreXmlConfiguration()
+            // Disable default message interpolation which scans classpath and required runtime dependency
+            .buildValidatorFactory()
```

Comment:
do we need `.messageInterpolator(new ParameterMessageInterpolator())` here?

Reply:
Yeah, my bad, sorry. 

Had bad sleep today